exercise/templates/exercise/exercise_plain.html

```diff
@@ -87,24 +87,34 @@
                             </li>
                             {% if not exercise.category.confirm_the_level %}
                                 <li class="navbar-text">
-                                    {% if not exercise.is_closed %}
-                                        <small>
-                                            {% if exercise.points_to_pass > 0 %}
-                                            {% blocktrans with points=exercise.points_to_pass %}
-                                                {{ points }} points required to pass.
-                                            {% endblocktrans %}
-                                            {% endif %}
-                                            {% if exercise.is_late_submission_open %}
-                                                {% trans "Late submission deadline" %} {{ module.late_submission_deadline }}
-                                                {% if module.late_submission_penalty > 0 %}
-                                                (-{{ module.late_submission_penalty|percent }}%)
+                                      <small>
+                                          {% if exercise.points_to_pass > 0 %}
+                                          {% blocktrans with points=exercise.points_to_pass %}
+                                              {{ points }} points required to pass.
+                                          {% endblocktrans %}
+                                          {% endif %}
+                                          {% if exercise|deviation_deadline:profile %}
+                                              {% trans "Extended deadline" %} {{ exercise|deviation_deadline:profile }}
```

Comment:
Like I commented in the other HTML template, it is not good to query the database multiple times for the same deviation. The `with` template tag can be used to create a template variable that holds the return value of `exercise|deviation_deadline:profile`.